.circleci/config.yml

```diff
@@ -178,6 +178,7 @@ jobs:
           when: always
           command: |
             hubploy deploy gcp-uscentral1b pangeo-deploy ${CIRCLE_BRANCH} --cleanup-on-fail --namespace ${CIRCLE_BRANCH}
+            helm upgrade --wait --install metrics stable/prometheus-operator --namespace=metrics -f metrics/values.yaml
```

Comment:
This *might* fail CI. If it does I'll followup with a PR removing it and we'll need to manually update things when the dashboards change.